deps/cloudxr/webxr_client/src/index.html

```
@@ -17,1146 +17,1468 @@
+                  placeholder="Leave blank for factor mode, or set >= 2"
+                  spellcheck="false"
+                  value=""
+                />
+                <div
+                  id="reprojectionGridRowsValidationMessage"
+                  class="config-text"
+                  aria-live="polite"
+                ></div>
+                <div class="config-text">
+                  Grid rules: leave either field blank to use factor mode, or
+                  set both to integers >= 2 for explicit mesh resolution.
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)